components/Footer/index.js

```diff
@@ -6,19 +6,73 @@ import React from 'react';
 import {Link} from 'react-router';
 
 class Footer extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state = {msg: '', email: '', success: ''};
+  }
+
+  onSubmit(e){
+    e.preventDefault();
+    const reg = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    if (!reg.test(this.state.email)){
+      this.setState({msg: 'Please enter a valid email address.'});
+    } else {
+      fetch('http://gettoknow.skookum.com/acton/eform/17086/0005/d-ext-0001', {
```

Comment:
instead of using fetch, can you set this up the same way the current contact form works?
https://github.com/Skookum/skookum.com/blob/master/components/ContactForm/index.js#L69
In this case:
```
var aoCAP = {
        aid: '17086',
        fid: '0005',
        did: 'd-0001',
        server: 'gettoknow.skookum.com',
        formId: 'form_0005',
      };

You'll also need to import `import {AoProcessForm} from 'actOn';`
      AoProcessForm(e.target, aoCAP);
```